pkgs/top-level/all-packages.nix

```diff
@@ -27154,6 +27154,8 @@ with pkgs;
 
   postgresqlTestHook = callPackage ../build-support/setup-hooks/postgresql-test-hook { };
 
+  postgrest = haskell.lib.compose.justStaticExecutables haskellPackages.postgrest;
```

Comment:
I would prefer to enableSeparateBinOutput in configuration-nix.nix and then use postgrest = haskellPackages.postgrest.bin; here.
Your solution is valid and what has historically been done, but I prefer the output based solution because it makes it possible to share the same derivation between haskellPackages and this top level attribute.

Reply:
I *think* I understand the difference. Done.